agent/mibgroup/if-mib/data_access/interface.c

```diff
@@ -782,6 +790,40 @@ netsnmp_access_interface_entry_overrides(netsnmp_interface_entry *entry)
     }
 }
 
+/**---------------------------------------------------------------------*/
+/*
+ * include_ifmib_iface_prefix config token
+ *
+ *     If there is an iface name match, we return 1 for True
+ *     otherwise return 0 for False.
+ *     If there are no ifaces defined at all, return 1 so that the
+ *     default behavior is to include all ifaces (include everything).
+ *     (Note: including at least one iface name means we will only include
+ *     those iface names and exclude all others.)
+ *
+ */
+int netsnmp_access_interface_include(const char * name)
+{
+    netsnmp_include_if_list * if_ptr;
+
+    netsnmp_assert(1 == _access_interface_init);
```

Comment:
Please put the constant at the right of "==". All C compilers today issue a warning when using an assignment in an if-condition.

Reply:
ok.